xorg/x11-forwarding.txt

```diff
@@ -0,0 +1,117 @@
+X11-FORWARDING
+________________________________________________________________________________
+
+X11-Forwarding is a secure shell feature, which allows one to forward/tunnel
+X11 connections through an existing SSH shell session. This is used to run X11 
```

Comment:
`SSH shell` -> `SSH` ("Secure Shell Shell"?) This might be fine to keep as-is though. (kinda similar to "ATM Machine").